OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1869 @@
+
+// =============================================================================
+// Scene-backed integration tests (entity_utils with real Scene + Entity)
+// =============================================================================
+
+class LuaSceneTest : public ::testing::Test
+{
+  protected:
+    sol::state lua;
+    Ref<Scene> scene;
+
+    void SetUp() override
```

Clang (14.0.6)

[warning] method 'SetUp' can be made static

(readability-convert-member-functions-to-static)